services/openuserjs/openuserjs-license.tester.js

```diff
@@ -0,0 +1,20 @@
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('License (valid)')
+  .get('/l/DenverCoder1/Unedit_and_Undelete_for_Reddit.json')
+  .expectBadge({
+    label: 'license',
+    message: 'MIT',
+  })
+
+t.create('Licenses (multiple)')
+  .get('/l/Marti/oujs_-_Meta_View.json')
```

Comment:
This should be fine since I'm the Admin and co-owner of the site. This is my only glory I get these days :smile_cat: .

Reply:
Tell you what... if you *(and Shields)* prefer I can create some Unit Tests *(dummy metadata blocks only)* under https://openuserjs.org/users/author so it will make the code more readable. That is also a reserved account and shouldn't be going anywhere. I can also create dummy issues for the counts.

Just give me some names *(or I can do it)* and initial values.

Reply:
I was thinking of making some fake scripts on my own account, but if you want to officially create some, that would work great as well.

For testing the metrics, it just needs to have a positive number (eg. non-zero number of installs, non-zero number of issues, etc.)

Reply:
> it just needs to have a positive number 

It's a full account, just no auth, so anyone can affect the values... .however I should create the base issues for test scripts with my Admin account so others above my ranking know why it exists. Anyone will be able to up/downrate it too but I might consider restricting that in the source... doesn't mean it goes away just not list-able at this point in time if too far down the rating.

Reply:
There you go...  https://openuserjs.org/users/author/scripts left the fork of Meta View as negative rating. If you want different numbers on install count tap the install it a few random times *(but don't go overboard)*. If you want more generic script names tell me. I left it so it can showcase what you have done. 

For `NotAScript` that is absent for that Shields test.

Reply:
Looks good. 

Maybe there could be one to replace the "no version found" unit test as well for consistency.

This is currently the only one I can't replace with the "author" account yet https://openuserjs.org/scripts/DenverCoder1/Example_no_version

Reply:
Forked and done.